src/date-picker.css

```diff
@@ -9,10 +9,12 @@
 }
 
 .react-datepicker__input-container {
-  font-size: 1rem;
-  padding-left: 1rem;
-  padding-right: 1rem;
+  font-size: 0.8rem;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  /* margin: 0 0 0 2rem; */
```

Comment:
can you remove the lines instead of commenting them out, please?

Reply:
You bet -- done.